### DISCUSSION TITLE
JetPack 6.0 & Humble

### DISCUSSION THREAD

**Discussion by bblumberg** [NVlabs/curobo#129] [Software Q&A] [answered]
Hi
I am working on an AGX ORIN with Jetpack 6.0. I modified the aarch64 docker file so (a) it used dustynv/l4t-pytorch:r36.2.0 from jetson-containers (CUDA 12.2 and pytorch 12.1), and (b) so it uses Humble as opposed to Noetic. 

So my question is whether I have entered a world of pain, or a world that no one has tested to know what pain lurks, or a world that should work. Thanks for any guidance.

Congrats, curobo is very impressive work!

**Comment by balakumar-s** [comment]
Thanks. As long as you can run Pytorch and warp, curobo should also work. Though we haven't tested on JetPack 6. Let us know if you have any issues and we can help.

**Comment by bblumberg** [comment]
Hi, thanks for the quick reply. I did get hit by the wp.index() breaking change that was referenced in one of the issues, but I think that is a positive sign. Is there a branch that has the fix? Thanks again!!!

bb

**Reply by balakumar-s** [reply]
We are working on a fix, this should be out in 8 hours.

**Comment by bblumberg** [comment]
Awesome. Many thanks!

**Comment by balakumar-s** [comment, marked as answer]
We pushed a fix for wp.index. Pull the latest commit and it should work.

**Comment by bblumberg** [comment]
Thanks so much for the quick turnaround. I was able to run 2 of the
examples successfully so positive evidence that it works. Once again, this
is using a modified dockerfile for aarch64 that uses the dustynv pytorch
image, and pulling in ros-humble-ros-base (noetic doesn't build on 22.04).

bb
